client/lcd/lcd_test.go

```diff
@@ -231,7 +231,7 @@ func TestCoinMultiSendGenerateOnly(t *testing.T) {
 	var stdTx auth.StdTx
 	require.Nil(t, cdc.UnmarshalJSON([]byte(body), &stdTx))
 	require.Equal(t, len(stdTx.Msgs), 1)
-	require.Equal(t, stdTx.GetMsgs()[0].Route(), "bank")
+	require.Equal(t, stdTx.GetMsgs()[0].Route(), bank.RouterKey)
```

Comment:
Doesn't the `AppModule` interface have a `Name()`? Should we be using that instead?

Reply:
Yeah I can just kinda change _everything_ to be using the module name, I've currently left things moreless how their structured which is define other constants to the name and then pass those these ways... I'm not sure that really makes sense - probably everything should just be switched to the module name aye?